sdk/identity/test-resources.bicep

```diff
@@ -58,6 +58,9 @@ output IDENTITY_USER_DEFINED_CLIENT_ID string = deployMIResources? managedIdenti
 output IDENTITY_USER_DEFINED_IDENTITY_NAME string = deployMIResources? managedIdentityModule.outputs.IdentityUserDefinedIdentityName : ''
 output IDENTITY_STORAGE_NAME_1 string = deployMIResources? managedIdentityModule.outputs.IdentityStorageName1 : ''
 output IDENTITY_STORAGE_NAME_2 string = deployMIResources? managedIdentityModule.outputs.IdentityStorageName2 : ''
+output IDENTITY_STORAGE_ID_1 string = deployMIResources? managedIdentityModule.outputs.IdentityStorageId1 : ''
```

Comment:
it would help to distinguish these by naming one of them `IDENTITY_STORAGE_ID_USER_ASSIGNED` or similar